### ISSUE TITLE
Benchmarking on CIFAR-100

### ISSUE THREAD

**Issue by abhinavagarwalla** [leftthomas/SimCLR#6]
I adapted this repo to work on CIFAR-100. I noticed that the accuracy computed using KNN varies greatly between subsequent epochs from around 40% to 1%. I believe that the learning rate of 1e-3 is very reasonable. Did you experiment with this dataset? Any ideas on what might be different?
Thanks!

**Comment by leftthomas** [comment]
@abhinavagarwalla I haven't test with this dataset, you could try the hyper-parameters which stated in the paper
